.github/workflows/R-CMD-check.yaml

```diff
@@ -74,5 +74,5 @@ jobs:
           needs: check
 
       - uses: r-lib/actions/check-r-package@v2
-        with:
-          error-on: '"note"'
+        # with:
+        #   error-on: '"note"'
```

Comment:
😬


        happy to test this out, we may revert it slows us down a lot

_Originally posted by @MichaelChirico in https://github.com/r-lib/lintr/pull/1621#pullrequestreview-1132006330_